### ISSUE TITLE
Extension remembers removed remote

### ISSUE THREAD

**Issue by manueldeljesus** [germanztz/gnome-shell-extension-rclone-manager#18]
I have configured only one remote. I have been playing with the extension, trying to add another remote. After that, I removed the remote completely. However, every restart, the extension is complaining about that remote -that I removed- not being available.

I have been looking for any config file that may have kept the info, but found none.

Any suggestion as to how can I deal with the issue or at least get more info so that you can diagnose it?

**Comment by manueldeljesus** [comment]
Just to make things clear. The remote does not appear neither in the extension nor when launching the rclone command. However, when starting the session, a message errors pops up in GNOME and the extension shows an alert triangle as an icon.

**Comment by germanztz** [comment]
Hi Manuel, please try the reset to defaults options in the settings screen,
I'll check if this is an issue anyways

Thanks

**Comment by manueldeljesus** [comment]
I have reset the defaults but it is still creating the folder.

I unchecked the option to sync on start, and that removed the error message. But it is still creating a folder for the removed remote. Where is the information stored? How does the extension now that I once created that remote?

I have installed and removed the extension something like three or four times testing things, and I cannot find where that information may be stored.
